tabbycat/draw/manager.py

```diff
@@ -220,7 +265,16 @@ def get_teams(self):
                 setattr(team, pullup_metric, standing.metrics[pullup_metric])
             ranked.append(team)
 
-        return ranked
+        n_byes = self.n_byes(len(ranked))
+        if n_byes:
+            if self.round.tournament.pref('bye_team_selection') == 'random':
+                byes = []
+                for i in range(n_byes):
+                    byes.append(ranked.pop(random.randrange(len(ranked))))
```

Comment:
Another option could be to use `random.sample(n_byes)` (though you'd still have to loop through the result to `pop` them from `ranked`) (and similarly in `SeededDrawManager`).